src/main/kotlin/com/google/container/tools/init/PluginInitComponent.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.init
+
+import com.google.container.tools.core.PluginInfo
+import com.google.container.tools.core.analytics.UsageTrackerManagerService
+import com.intellij.ide.plugins.PluginManager
+import com.intellij.openapi.components.BaseComponent
+import com.intellij.openapi.extensions.PluginId
+
+/**
+ * Performs initialization tasks when the plugin is loaded.
+ */
+class PluginInitComponent : BaseComponent {
+
+    override fun initComponent() {
+        val isGctInstalled =
+            PluginManager.isPluginInstalled(PluginId.getId(PluginInfo.GOOGLE_CLOUD_TOOLS_PLUGIN_ID))
```

Comment:
Does the loading order guarantee we can check this condition regardless of which plugin gets loaded first?

Reply:
I believe so. This check is that the plugin is "installed" not that its "loaded". So a plugin is considered installed even before load. All my testing confirms this.